Dockerfile

```diff
@@ -8,6 +8,8 @@ RUN npm ci
 
 RUN npm run build
 
+RUN npm run migrate
```

Comment:
i think migrations need to be run post-build as part of the GHA workflow, otherwise there's no db to connect to

Reply:
Ahh, you're right! I was thinking the `depends_on` would ensure that the database was running, I forgot that images are fetched/built before any container is started.